.github/workflows/io-tests.yaml

```diff
@@ -0,0 +1,108 @@
+
+    - name: Install dependencies
+      run: npm ci
+    
+    - name: Run tests
+      run: npm test || if [ $? -eq 10 ]; then exit 0; else exit $?; fi
+
+    - name: Install AIO CLI
+      if: success() && github.event_name == 'pull_request'
+      run: npm install -g @adobe/aio-cli
+
+    - name: Deploy action
+      if: success() && github.event_name == 'pull_request'
+      env:
+        AIO_runtime_namespace: ${{ secrets.AIO_NS_STAGE }}
+        AIO_APIHOST: ${{ secrets.AIO_APIHOST }}
+        AIO_runtime_auth: ${{ secrets.AIO_AUTH_STAGE }}
+        AOS_API_KEY: ${{ secrets.AOS_API_KEY }}
+        AOS_URL: ${{ secrets.AOS_URL }}
+        ODIN_CDN_ENDPOINT: ${{ secrets.ODIN_CDN_ENDPOINT }}
+        ODIN_ORIGIN_ENDPOINT: ${{ secrets.ODIN_ORIGIN_ENDPOINT }}
+        OST_WRITE_API_KEY: ${{ secrets.OST_WRITE_API_KEY }}
+        WCS_CDN_ENDPOINT: ${{ secrets.WCS_CDN_ENDPOINT }}
+        WCS_ORIGIN_ENDPOINT: ${{ secrets.WCS_ORIGIN_ENDPOINT }}
+      run: |
+        # Check for missing secrets
+        missing_secrets=()
+        
+        [ -z "$AIO_runtime_namespace" ] && missing_secrets+=("AIO_runtime_namespace")
+        [ -z "$AIO_APIHOST" ] && missing_secrets+=("AIO_APIHOST")
+        [ -z "$AIO_runtime_auth" ] && missing_secrets+=("AIO_runtime_auth")
+        [ -z "$AOS_API_KEY" ] && missing_secrets+=("AOS_API_KEY")
+        [ -z "$AOS_URL" ] && missing_secrets+=("AOS_URL")
+        [ -z "$ODIN_CDN_ENDPOINT" ] && missing_secrets+=("ODIN_CDN_ENDPOINT")
+        [ -z "$ODIN_ORIGIN_ENDPOINT" ] && missing_secrets+=("ODIN_ORIGIN_ENDPOINT")
+        [ -z "$OST_WRITE_API_KEY" ] && missing_secrets+=("OST_WRITE_API_KEY")
+        [ -z "$WCS_CDN_ENDPOINT" ] && missing_secrets+=("WCS_CDN_ENDPOINT")
+        [ -z "$WCS_ORIGIN_ENDPOINT" ] && missing_secrets+=("WCS_ORIGIN_ENDPOINT")
+        
+        if [ ${#missing_secrets[@]} -ne 0 ]; then
+          echo "Error: The following required secrets are empty:"
+          printf '%s\n' "${missing_secrets[@]}"
+          exit 1
+        fi
+
+        # Configure and deploy
+        aio config set AIO_runtime_namespace "$AIO_runtime_namespace"
+        aio config set AIO_runtime_auth "$AIO_runtime_auth"
+        aio config set AIO_APIHOST "$AIO_APIHOST"
+        aio config set AOS_API_KEY "$AOS_API_KEY"
+        aio config set AOS_URL "$AOS_URL"
+        aio config set ODIN_CDN_ENDPOINT "$ODIN_CDN_ENDPOINT"
+        aio config set ODIN_ORIGIN_ENDPOINT "$ODIN_ORIGIN_ENDPOINT"
+        aio config set OST_WRITE_API_KEY "$OST_WRITE_API_KEY"
+        aio config set WCS_CDN_ENDPOINT "$WCS_CDN_ENDPOINT"
+        aio config set WCS_ORIGIN_ENDPOINT "$WCS_ORIGIN_ENDPOINT"
+        aio app deploy
+
+    - name: Run Playwright Tests
+      if: success() && github.event_name == 'pull_request'
+      env:
+        TEST_URL: https://stage--milo--adobecom.hlx.live/libs/features/mas/docs/ccd.html?mas-io-url=https://14257-merchatscale-Stage.adobeioruntime.net/api/v1/web/MerchAtScale
```

Comment:
mmm shouldn't we have a `14257-merchatscale-<ldap>` instead of `14257-merchatscale-Stage`?